07-modeling.Rmd

```diff
@@ -15,8 +17,10 @@ $$Y_i=\beta_1 X_i+\epsilon_i$$
 
 it can be specified as `y~x-1`. Formula notation details in R can be found in the help file for formula^[Use `help(formula)` in R or find the documentation online at https://stat.ethz.ch/R-manual/R-devel/library/stats/html/formula.html]. A quick overview of the common formula notation is in the following table:
 
+Table: Common symbols in formula notation
+
 Symbol | Example | Meaning
--|-|-
+-|-|- 
```

Comment:
Need to adjust the column widths.  All 3 are the same size now.  But the first two should be much smaller than the last one (at least in the PDF).